validator/remote/src/main/java/tech/pegasys/teku/validator/remote/EventSourceBeaconChainEventAdapter.java

```diff
@@ -96,6 +125,113 @@ private HttpUrl createHeadEventSourceUrl(final HttpUrl endpoint) {
     return Preconditions.checkNotNull(eventSourceUrl);
   }
 
+  // connect to a failover event stream only if syncing status call throws
+  // an exception and there is a ready failover Beacon Node
+  @SuppressWarnings({"FutureReturnValueIgnored"})
```

Comment:
Played a bit to avoid the suppress
<img width="1084" alt="image" src="https://user-images.githubusercontent.com/15999009/179789275-8b7adb7d-704d-47c3-a69c-e128aaab827a.png">
Could be better if its equivalent :)

Reply:
That looks better and more readable, implemented.